python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -109,12 +109,11 @@ set_tests_properties(test_parallel_executor_fetch_feed PROPERTIES TIMEOUT 450)
 py_test_modules(test_parallel_executor_transformer MODULES test_parallel_executor_transformer SERIAL)
 if(NOT APPLE)
     py_test_modules(test_image_classification_resnet MODULES test_image_classification_resnet SERIAL)
```

Comment:
Why move it out of `if(NOT APPLE)`?

Reply:
`test_parallel_executor_seresnext` can be runned in Mac.